scripts/Install-AllSoftware.ps1

```diff
@@ -0,0 +1,24 @@
+# 1. Make sure the Microsoft App Installer is installed:
+#    https://www.microsoft.com/en-us/p/app-installer/9nblggh4nns1
+# 2. Run this script as administrator. If running scripts is blocked, you can temporarily unblock them by running 
+	Set-ExecutionPolicy -ExecutionPolicy Unrestricted -Scope Process
+
+Write-Output "Installing apps"
+$apps = @(
+	@{name = "GitHub.GitLFS" },
```

Comment:
Is GitLFS still required?

Reply:
We shouldn't need it anymore since we no longer have any GitLFS files.